buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/publish/PublishingPlugin.groovy

```diff
@@ -121,10 +127,27 @@ class PublishingPlugin implements Plugin<Project> {
                             publisher.decorate(sub, it)
                         }
                     }
+
+                    // Add changed projects to publishChangedToBuildDir.
+                    if (changedProjects.contains(sub)) {
+                        publishChangedToBuildDir.dependsOn "$sub.path:publishMavenAarPublicationToBuildDirRepository"
+                        def dependencies = sub.configurations.implementation.dependencies
+                        for (def dep : dependencies) {
```

Comment:
What's the intent here? Why do we need to publish dependencies of the project? Also, if it is needed indeed, don't we also want to publish dependencies of dependencies recursively?